src/components/Registration/DefineRegistrationSummary.vue

```diff
@@ -24,6 +24,22 @@
         </v-row>
       </article>
 
+      <!-- Business Type Confirm -->
+      <template v-if="isTypePartnership">
+        <v-divider class="mx-6" />
+
+        <article class="section-container">
+          <v-row no-gutters>
+            <v-col cols="12" sm="3" class="pr-4">
+              <label>Business Type</label>
+            </v-col>
+            <v-col cols="12" sm="9">
+              <span>{{ businessTypeConfirm || false}}</span>
+            </v-col>
+          </v-row>
+        </article>
+      </template>
+
```

Comment:
I added the above template block in this file in my latest commit, because the unfinished steps/scroll down mechanics didn't work properly. I am not sure whether I should add this block though.

Reply:
Lines 28-41?

The template + v-if seems correct to me since this feature only applies to GPs.

PS I think this component (a subcomponent of RegistrationReviewConfirm page/view) doesn't have the error scrolling mechanism.

Reply:
ACTUALLY, I think this checkbox doesn't display on the review page.